### ISSUE TITLE
after option on multiple="checkbox" is positioned wrong

### ISSUE THREAD

**Issue by t1mmen** [slywalker/TwitterBootstrap#18]
This:

``` php
<?php echo $this->BootstrapForm->input('sample4_0', array(
    'label' => 'Checkboxes',
    'multiple' => 'checkbox',
    'after' => 'Note:  This is positioned incorrectly',
    'options' => array(
        1 => '1',
        2 => '2',
        3 => '3',
        4 => '4',
        5 => '5',
    ),
)); ?>
```

Generates this:
http://cl.ly/0N2w201x421H1G1H3c1r

But it should look more like:
http://cl.ly/3p020L3J3G300u1v1a1F

**Comment by slywalker** [comment]
It fixed 1.1.0.

**Comment by t1mmen** [comment]
Thanks! :)
